package.json

```diff
@@ -21,13 +21,14 @@
     "preinstall": "node scripts/preinstall.js",
     "pretest": "npm run check-format",
     "posttest": "eslint . --ext .ts --config .eslintrc",
-    "prepack": "rm -rf lib && tsc -b && oclif-dev readme",
-    "start": "./bin/run",
+    "prepack": "rm -rf lib tsconfig.tsbuildinfo && tsc -b && oclif-dev readme",
+    "start": "npm run prepack && ./bin/run",
     "test": "cross-env NODE_ENV=test nyc --extension .ts mocha --forbid-only \"{src,test}/**/*.test.ts\"",
     "prod_test": "cross-env NODE_ENV=test nyc --extension .ts mocha --forbid-only \"prod_test/**/*.test.ts\"",
     "version": "oclif-dev readme && git add README.md",
     "format": "prettier --write .",
-    "check-format": "prettier --check ."
+    "check-format": "prettier --check .",
+    "pkg": "npm run prepack && pkg --out-path dist/ . --compress brotli"
```

Comment:
`out-path` is not necessary since it's already specified in `pkg` field in this JSON file.

Reply:
Whoopsie, should be fixed in the latest commit!